src/app/courses-list/courses-list.component.html

```diff
@@ -1,7 +1,21 @@
 <div class="courses-list">
-    <app-search></app-search>
-        <app-course *ngFor="let course of courses$ | async" [course]="course" (removeItem)="onNotify($event)"></app-course>
-    <div class="load-more">
-        <button (click)="loadMore()">Load More</button>
-    </div>
+    <app-search (notify)="onSearchNotify($event)"></app-search>
+<!--    @TODO: get rid of two subscriptions to courses$-->
```

Comment:
it is ok to have several async pipe subscriptions in the template. So what u have to worry about - your stream should be shared and return last emit to each subscriber.
U can use publishReplay() and refCount(1) rxjs operators